lib/pypiCoordinatesMapper.js

```diff
@@ -21,19 +22,16 @@ class PypiCoordinatesMapper {
   }
 
   async _resolve(coordinates) {
+    if (coordinates.name === '..') return null
```

Comment:
I was wondering if we want a more generalized function to sanitize coordinates? But also fine as is 👍🏼

Reply:
I thought about this as well: using URL object and the check `/../` in the final url can be refactored into a function.  I will do the refactoring as a separate PR.  The check for `/` in coordinates.name is specific to coordinates, which maybe harder to refactor.